src/components/tab-title/tab-title.e2e.ts

```diff
@@ -14,8 +17,8 @@ describe("calcite-tab-title", () => {
     const page = await newE2EPage();
     await page.setContent(`<calcite-tab-title icon-start='plus'>Text</calcite-tab-title>`);
     const element = await page.find("calcite-tab-title");
-    const iconStart = await page.find("calcite-tab-title >>> .calcite-tab-title--icon.icon-start");
-    const iconEnd = await page.find("calcite-tab-title >>> .calcite-tab-title--icon.icon-end");
+    const iconStart = await page.find(iconStartHtml);
+    const iconEnd = await page.find(iconEndHtml);
```

Comment:
Thanks for the cleanup! :broom:

Reply:
Yep, been working with design on this. I'll run it through them again to make sure.